torchvision/models/vgg.py

```diff
@@ -47,16 +47,15 @@ def forward(self, x):
     def _initialize_weights(self):
         for m in self.modules():
             if isinstance(m, nn.Conv2d):
-                n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
-                m.weight.data.normal_(0, math.sqrt(2. / n))
+                nn.init.kaiming_normal_(m.weight, mode="fan_out")
```

Comment:
same here, I think it should have the `nonlinearity='relu'` here as well, right?